### ISSUE TITLE
Missing red dot

### ISSUE THREAD

**Issue by shami3** [SlendyMilky/CFF-Clock#1]
The “seconds” hand is missing the red dot as per SBB clock photo (https://www.uhrenschmuck24.ch/mondaine-wall-clock-a990-clock-16sbb.html)

**Comment by ghost** [comment]
The image you provide represents the wall clock, which is not the same as the one we see in a station.
In a station, there is no dot in the middle. It can clearly be seen in the photos of this article from swissinfo:
https://www.swissinfo.ch/fre/finance/jubil%C3%A9_la-c%C3%A9l%C3%A9brissime-horloge-cff-a-75-ans/44690372

**Comment by shami3** [comment]
I noticed just now 😄, it could be provided as option anyway. There is also a small white dot in some SBB clocks that could be added, but it doesn’t appear in all the photos so I’m not sure

**Comment by SlendyMilky** [comment]
> I noticed just now 😄, it could be provided as option anyway. There is also a small white dot in some SBB clocks that could be added, but it doesn’t appear in all the photos so I’m not sure

Yeah that could be an option, at the moment i prefer to make sure the clock appear correctly on any screen.
